[2024-11-12T04:26:36.149+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-12T04:26:36.191+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ETL_To_Data_Warehouse.clear_data_to_stgBenef manual__2024-11-12T04:26:07.393313+00:00 [queued]>
[2024-11-12T04:26:36.202+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ETL_To_Data_Warehouse.clear_data_to_stgBenef manual__2024-11-12T04:26:07.393313+00:00 [queued]>
[2024-11-12T04:26:36.203+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-12T04:26:36.218+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): clear_data_to_stgBenef> on 2024-11-12 04:26:07.393313+00:00
[2024-11-12T04:26:36.226+0000] {standard_task_runner.py:63} INFO - Started process 39968 to run task
[2024-11-12T04:26:36.229+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ETL_To_Data_Warehouse', 'clear_data_to_stgBenef', 'manual__2024-11-12T04:26:07.393313+00:00', '--job-id', '680', '--raw', '--subdir', 'DAGS_FOLDER/ETL_LDTBXH.dag.py', '--cfg-path', '/tmp/tmphz4tck9u']
[2024-11-12T04:26:36.231+0000] {standard_task_runner.py:91} INFO - Job 680: Subtask clear_data_to_stgBenef
[2024-11-12T04:26:36.285+0000] {task_command.py:426} INFO - Running <TaskInstance: ETL_To_Data_Warehouse.clear_data_to_stgBenef manual__2024-11-12T04:26:07.393313+00:00 [running]> on host 3d0ab7361361
[2024-11-12T04:26:36.673+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ETL_To_Data_Warehouse' AIRFLOW_CTX_TASK_ID='clear_data_to_stgBenef' AIRFLOW_CTX_EXECUTION_DATE='2024-11-12T04:26:07.393313+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-12T04:26:07.393313+00:00'
[2024-11-12T04:26:36.674+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-12T04:26:36.728+0000] {logging_mixin.py:188} WARNING - /opt/***/plugins/NCC/clear_data_stgBenef.py:16 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2024-11-12T04:26:40.263+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-12T04:26:40.264+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-12T04:26:40.276+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ETL_To_Data_Warehouse, task_id=clear_data_to_stgBenef, run_id=manual__2024-11-12T04:26:07.393313+00:00, execution_date=20241112T042607, start_date=20241112T042636, end_date=20241112T042640
[2024-11-12T04:26:40.334+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-11-12T04:26:40.548+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-12T04:26:40.550+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
